test/e2e/framework.go

```diff
@@ -103,3 +103,47 @@ func (f *Framework) WaitForPodRunning(podName string) error {
 func (f *Framework) TestContainerOutput(scenarioName string, pod *api.Pod, expectedOutput []string) {
 	testContainerOutputInNamespace(scenarioName, f.Client, pod, expectedOutput, f.Namespace.Name)
 }
+
+func (f *Framework) WaitForEndpoints(serviceName string) error {
```

Comment:
Document what this does.  The name seems to indicate it waits for all to be ok, but the implementation just waits for one to be ok.